.github/workflows/add-issues-to-project.yml

```diff
@@ -1,16 +1,43 @@
 # GitHub action that automatically adds newly filed issues to the Lit project.
-# See https://github.com/leonsteinhaeuser/project-beta-automations for documentation.
+#
+# Based on the example at
+# https://docs.github.com/en/issues/trying-out-the-new-projects-experience/automating-projects#example-workflow-authenticating-with-a-personal-access-token
 on:
   issues:
     types: [opened]
 jobs:
   add-issue-to-project:
     runs-on: ubuntu-latest
     steps:
-      - name: Add issue to Lit project
-        uses: leonsteinhaeuser/project-beta-automations@v1.1.0
-        with:
-          gh_token: ${{ secrets.LIT_ROBOT_AUTOMATION_PAT }}
-          organization: lit
-          project_id: 4
-          resource_node_id: ${{ github.event.issue.node_id }}
+      # Finds the global entity ID for the project and adds it to an environment variable
+      # for the next step to consume.
+      - name: Get project ID
```

Comment:
Is the global entity ID for a project static? I.e. could be stored directly in secrets?

Reply:
It is, and it's not a secret so it could also just be inlined here. However it's a weird identifier that's a bit hard to find because you have to do an API query to get it, so it's more convenient to just query for it directly here. That way this script can be updated easily, used in other contexts etc.